package.json

```diff
@@ -1,37 +1,58 @@
 {
   "name": "mustard",
   "version": "0.0.1",
-  "description": "A tagger tool for music files and simpe player",
+  "description": "A tagger tool for music files and simple player",
   "main": "main.js",
+  "homepage": "https://github.com/angrykoala/mustard#readme",
+  "dependencies": {
+    "browser-id3-writer": "^3.0.2",
+    "jquery": "^3.1.1",
+    "musicmetadata": "^2.0.5",
+    "vue": "^2.1.8"
+  },
+  "devDependencies": {
+    "electron": "^1.4.14",
+    "electron-builder": "^19.41.1",
+  },
   "scripts": {
     "start": "electron .",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "jshint": "jshint index.js app/*.js",
```

Comment:
Why did you remove jshint?

Reply:
Misunderstood one of your previous questions. Already fixed